lib/awful/util.lua

```diff
@@ -344,6 +344,13 @@ function util.get_rectangle_in_direction(dir, recttbl, cur)
     return grect.get_in_direction(dir, recttbl, cur)
 end
 
+-- @deprecated awful.util.pread
+function util.pread ()
+    util.deprecate("Use io.popen() directly or look at awful.spawn.easy_async() "
+            .. "for an asynchronous alternative")
+    return ""
+end
```

Comment:
This kind-of-conflicts with the same thing done in `lib/awful/init.lua`. No time to test right now, but would it also work if there were no actual lua-function after the doc comment?
(Actually: Does the proposed patch work? The doc comment begins with `--` but I thought `---` is needed?)
(Also: Typo in the commit message: awEful)